.github/workflows/cd-release-npm.yml

```diff
@@ -3,7 +3,7 @@ name: Release NPM Packages
 on:
   workflow_dispatch:
   schedule:
-    - cron: '0 0 * * 1'
+    - cron: '0 7 * * 0-4'
```

Comment:
Verifying that this is correct:

> At 07:00 on every day-of-week from Sunday through Thursday

https://crontab.guru/#0_7_*_*_0-4

Reply:
Yeah this just goes back to what we had previously.